File: benchmark/results/java-multilang/run-003/b-baseline/output/com/metaengine/demo/ordering/services/OrderService.java
package com.metaengine.demo.ordering.services;

import com.metaengine.demo.ordering.aggregates.Order;
import java.util.List;

/** OrderService domain service for the ordering domain. */
public class OrderService {
    /** Create a new Order from a partial input. */
    public Order create(Order input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find an Order by its identifier. May return null. */
    public Order findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List Orders up to the given limit. */
    public List<Order> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete an Order by its identifier. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/python-multilang/run-001/a-mcp/output/ordering/services/order_service.py
from ordering.aggregates.order import Order

"""OrderService service."""
class OrderService:
    def create(self, input: dict) -> Order:
        raise NotImplementedError('not implemented')
    def find_by_id(self, id: str) -> Order | None:
        raise NotImplementedError('not implemented')
    def list(self, limit: int) -> list[Order]:
        raise NotImplementedError('not implemented')
    def delete(self, id: str) -> None:
        raise NotImplementedError('not implemented')

